nixos/modules/services/x11/xserver.nix

```diff
@@ -523,6 +523,16 @@ in
         '';
       };
 
+      logFile = mkOption {
+        type = types.nullOr types.str;
+        # Log at the default verbosity level to stderr rather than /var/log/X.*.log.
+        default = "/dev/null";
+        example = "/var/log/Xorg.0.log";
+        description = ''
+          Controls the file Xorg logs to.
+        '';
+      };
+
```

Comment:
There should be a tmpfiles rules along in this commit.

Reply:
Then I guess that not giving a tmpfiles rules is a sort of discouragement to set this option to something else.